eslint.config.js

```diff
@@ -95,18 +95,23 @@ export default tseslint.config(
         'error',
         { allow: ['arrowFunctions'] },
       ],
+      '@typescript-eslint/no-empty-object-type': [
+        'error',
+        { allowInterfaces: 'with-single-extends' }, // maybe we should turn this on in a new PR
+      ],
       '@typescript-eslint/no-empty-interface': 'off',
       '@typescript-eslint/no-explicit-any': 'off', // maybe we should turn this on in a new PR
       'no-extra-semi': 'off',
       '@typescript-eslint/no-extra-semi': 'off', // conflicts with prettier
       '@typescript-eslint/no-inferrable-types': 'off',
+      '@typescript-eslint/no-unused-expressions': 'off', // maybe we should turn this on in a new PR
```

Comment:
[`@typescript-eslint/no-unused-expressions`](https://v8--typescript-eslint.netlify.app/rules/no-unused-expressions) was promoted to the [`recommended` config](https://v8--typescript-eslint.netlify.app/users/configs/#recommended) in v8.